tools/acle_template.tex

```diff
@@ -0,0 +1,141 @@
+% Copyright (c) 2014--2021, John MacFarlane
+% Based on default.latex template released with Pandoc (www.pandoc.org).
+% License: BSD 3-clause
+% See `data/templates` in `COPYRIGHT` at https://github.com/jgm/pandoc
+
+\documentclass[$if(fontsize)$$fontsize$,$endif$$if(papersize)$$papersize$paper,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
+\usepackage{xcolor}
+\usepackage{hyperref}
+\usepackage{longtable,booktabs}
+\urlstyle{same}  % don't use monospace font for urls
+\setlength{\emergencystretch}{3em}  % prevent overfull lines
+\providecommand{\tightlist}{%
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+\usepackage{fancyhdr}
+\usepackage{etoolbox}
+
+$if(geometry)$
+\usepackage[$for(geometry)$$geometry$$sep$,$endfor$]{geometry}
+$endif$
+
+\makeatletter
+  \patchcmd{\f@nch@foot}{\rlap}{\color{$if(fontcolor)$$fontcolor$$else$black$endif$}\rlap}{}{}
+  \patchcmd{\footrule}{\hrule}{\color{$if(fontcolor)$$fontcolor$$else$black$endif$}\hrule}{}{}
+\makeatother
+
+\pagestyle{fancy}
+\fancyhf{}
+\fancyfoot[L]{\small \emph{Copyright © 2020-2021, Arm Limited and its affiliates. All rights reserved.}}
+\fancyfoot[R]{\small \thepage}
+\renewcommand{\footrulewidth}{0.4pt}
```

Comment:
We need to have the copyright footer on all pages (new chapters, tables of contents) but not the title page. To do that, it seems that you need to re-defined the style `plain` of `fancyheader` (hopefully the change will not apply to the title page, which, iirc, is using a style called `empty`). You can get some ideas on how to redefine the `plain` style here: https://tex.stackexchange.com/questions/117328/fancyhdr-does-not-apply-same-header-footer-on-chapter-and-non-chapter-pages